### DISCUSSION TITLE
Is there anyway of defining separate asset link for app.css and app.js

### DISCUSSION THREAD

**Discussion by charliex311** [opcodesio/log-viewer#229] [Q&A] [answered]
Hi there,
I have an application that dynamically loads the asset links. In simple: i cannot bind the asset link via .env file or point it towards the /public/.... Is there any way of setting asset links statically or define somewhere from where the Log Viewer can load the new link?

Thanks.

**Comment by arukompas** [comment, marked as answer]
Hey @charliex311 

You can publish the layout file which contains the links to the assets. If you host the assets somewhere else, and you cannot use the Laravel's `ASSET_URL` environment variable for some reason, just do this:

```
php artisan vendor:publish --tag=log-viewer-views
```

And then update the layout's code so that it loads the assets from where you need them to be loaded.

Hope this helps!
